<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTexture</class>
 <widget class="QDialog" name="DlgTexture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建纹理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>纹理宽度:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_width">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>128</number>
       </property>
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="value">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>纹理高度:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_height">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>128</number>
       </property>
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="value">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>字符间隔:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_spacing">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_create">
       <property name="text">
        <string>创建</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PreviewTextureWidget" name="w_preview" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>缩放:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spb_scale">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_page">
       <property name="text">
        <string>页(%1/%2)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_page">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>背景色:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorButton" name="btn_background">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">control/color_button.h</header>
  </customwidget>
  <customwidget>
   <class>PreviewTextureWidget</class>
   <extends>QWidget</extends>
   <header location="global">dialog/dlg_texture.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_ok</sender>
   <signal>clicked()</signal>
   <receiver>DlgTexture</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
